### DISCUSSION TITLE
How can I build a query for an array?

### DISCUSSION THREAD

**Discussion by fomvasss** [spatie/elasticsearch-query-builder#8] [Q&A]
Hello! How can I build a query for an array of items https://www.elastic.co/guide/en/elasticsearch/reference/7.16/query-dsl-terms-query.html? Analogue `whereIn` in Laravel

**Comment by webbaard** [comment]
it might be to late for the original poster but for future searches this might be what you're looking for 
`$builder->addQuery(TermsQuery::create('fieldName', $arrayOfTerms));`
